mypy/plugin.py

```diff
@@ -297,11 +297,6 @@ def class_type(self, self_type: Type) -> Type:
         """Generate type of first argument of class methods from type of self."""
         raise NotImplementedError
 
-    @abstractmethod
-    def builtin_type(self, fully_qualified_name: str) -> Instance:
```

Comment:
It looks like this has been marked as deprecated since 2018, so it seems okay to remove it now.